tests/core/test_context.py

```diff
@@ -214,3 +223,17 @@ def demo(named: Named) -> None:
 
     spec = empty_context._type_spec_for_function(demo)
     assert spec == {"named": (hasname_ref, None)}
+
+
+def test_str_default(empty_context: BlueskyContext, sim_motor: SynAxis):
+    movable_ref = empty_context._reference(Movable)
+
+    spec = empty_context._type_spec_for_function(has_default_str)
+    assert spec == {"m": (movable_ref, SIM_MOTOR_NAME)}
```

Comment:
These are going to fail if/when #185 gets merged

Reply:
Since #185 is now approved, shall we merge it and resolve the conflicts here?

Reply:
:+1:  from my side

Reply:
Now merged